apps/dapp/src/components/GlobalFooter.tsx

```diff
@@ -0,0 +1,128 @@
+import Image from 'next/image';
+import Link from 'next/link';
+import React from 'react';
+import DropLogoWhite from '../../public/drop-logo-white.svg';
+import { SubscribeForm } from './LandingPage/SubscribeForm';
+import { SocialLinks } from './SocialLinks';
+
+interface FooterLink {
+  label: string;
+  href: string;
+  isExternal?: boolean;
+}
+
+const FooterLinks1: FooterLink[] = [
+  {
+    label: 'Terms and Conditions',
+    href: '/page/terms-and-conditions',
+  },
+  {
+    label: 'Privacy Policy',
+    href: '/page/privacy-policy',
+  },
+  // {
+  //   label: "Technology FAQs",
+  //   href: "/",
+  // },
+];
+const FooterLinks2: FooterLink[] = [
+  {
+    label: 'Discord',
+    href: 'https://discord.com/invite/Wx8yAu3JKS',
+    isExternal: true,
+  },
+  {
+    label: 'X (Formerly Twitter)',
+    href: 'https://twitter.com/itsdropaudio',
+    isExternal: true,
+  },
+  {
+    label: 'Instagram',
+    href: 'https://www.instagram.com/itsdropaudio',
+    isExternal: true,
+  },
+  {
+    label: 'News/Blog',
+    href: '/blog',
+  },
+];
+
+const FooterLinks3: FooterLink[] = [
+  {
+    label: 'Learn',
+    href: '/blog/welcome-to-drop',

```

Comment:
maybe we change this to `/blog/welcome` bc/ people will be using a different brand